tests/js/utils.js

```diff
@@ -538,3 +538,14 @@ export const unexpectedSuccess = () => {
 		)
 	);
 };
+
+/**
+ * Returns a promise that resolves after the specified delay, defaulting to 1ms.
+ *
+ * @since n.e.x.t
+ *
+ * @param {number} [delay] The delay in milliseconds. Defaults to 1.
+ * @return {Promise} A promise that resolves after the specified delay.
+ */
+export const resolvingPromise = ( delay = 1 ) =>
```

Comment:
Is the delay particularly significant? I see in some tests we use `5` while others use the default. The current/closest use of this does not provide a delay at all https://github.com/google/site-kit-wp/blob/d0cd49c2d47d758ce7991fd7778a35bb755252a3/tests/js/setup-before-after.js#L63

Any reason not to use `undefined` as default here as well?

Secondarily, the name is a bit ambiguous. Maybe something like `waitForDelay` would be more appropriate? That would make the argument a bit more clear as well when used.